lua/lazyvim/plugins/extras/lang/php.lua

```diff
@@ -48,7 +48,7 @@ return {
     optional = true,
```

Comment:
I did a repo search for "php-debug-adapter" and its only occurrence is here, so it isn't added to mason's `ensure_installed` anywhere. Adding it as a spec here, it'll only be installed if both the `dap.core` and `lang.php` extras are installed.

```suggestion
    optional = true,
    specs = {
        {
            "mason-org/mason.nvim",
            opts = {
              ensure_installed = {
                "php-debug-adapter",
              },
            },
        },
    },
```